app/src/main/res/drawable/divider.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape android:shape="rectangle" xmlns:android="http://schemas.android.com/apk/res/android">
     <size android:height="1dp"/>
-    <solid android:color="@color/separator_on_surface"/>
```

Comment:
Where is this one used? Do we risk an undesirable regression with this change?

Reply:
The layouts effected looks as expected :)